lms/djangoapps/course_home_api/outline/v1/views.py

```diff
@@ -82,9 +100,16 @@ def get(self, request, *args, **kwargs):
 
         course_blocks = get_course_blocks(request.user, course_usage_key, transformers, include_completion=True)
 
+        dates_widget = {  # information for the dates widget
```

Comment:
I would still say you don't need the comment here at all. The variable name makes it pretty clear on what this information is.